tests/conftest.py

```
@@ -3,218 +3,152 @@
+    config.addinivalue_line("markers", "e2e: marks tests as E2E tests")
+    config.addinivalue_line("markers", "unit: marks tests as unit tests")
+    config.addinivalue_line("markers", "integration: marks tests as integration tests")
 
-
-# ============================================================================
-# 🔧 Optional: Pytest hooks for better logging
-# ============================================================================
-
-def pytest_sessionstart(session):
-    """Called at the very start of test session"""
+def pytest_sessionstart(session): # แก้จาก _session เป็น session
```

Ruff (0.14.10)

Unused function argument: `session`

(ARG001)